wcc-contentful/lib/wcc/contentful/configuration.rb

```diff
@@ -173,4 +193,25 @@ def validate!
       raise ArgumentError, "The job '#{job}' must be an instance of ActiveJob::Base or respond to :call"
     end
   end
+
+  def freeze
+    FrozenConfiguration.new(self)
+  end
+
+  class FrozenConfiguration
```

Comment:
should this define `frozen?` to be consistent?

Reply:
also in the "non-frozen" config class?

Reply:
Good idea

Reply:
https://github.com/watermarkchurch/wcc-contentful/pull/138